Source/Plugins/Statistics/DistributionAnalysisOps/DistributionAnalysisOps.h

```diff
@@ -61,7 +61,7 @@ class DistributionAnalysisOps
     virtual int calculateCorrelatedParameters(std::vector<std::vector<float> >& data, VectorOfFloatArray outputs) = 0;
 
     static void determineMaxAndMinValues(std::vector<float>& data, float& max, float& min);
-    static void determineBinNumbers(float& max, float& min, float& numbins, FloatArrayType::Pointer binnumbers);
+    static void determineBinNumbers(float& max, float& min, float& stepsize, FloatArrayType::Pointer binnumbers);
```

Comment:
These parameter names imply different things.  Are we sure this change is correct?